torchvision/csrc/cuda/ROIPool_cuda.cu

```diff
@@ -193,7 +197,11 @@ at::Tensor ROIPool_backward_cuda(const at::Tensor& grad,
          pooled_height,
          pooled_width,
          grad_input.data<scalar_t>(),
-         rois.data<scalar_t>());
+         rois.data<scalar_t>(),
+         n_stride,
```

Comment:
interesting, so this adds support for non-contiguous gradients as well.
Did you have cases where that was the case?
It makes me think that I should probably add a `.contiguous()` call to `ROIAlign` as well

Reply:
I encountered it when I was writing the tests since as a quick hack, I was just doing 
```
output = roi_pool(input, rois)
s = output.sum()
s.backward()
```
and this was giving me issues because the gradient was a Tensor with `numel = 1` and all strides set to 0.

The current approach I implemented was more generic and I am happy to add this to ROIAlign as well once ROIPooling is merged.

Reply:
I see.
Yes, having the strides handling is definitely more general, but as a first step I'd just add the `.contiguous()` call, and later on add support for strides.

Reply:
Just to be sure, I should be calling `contiguous()` on the `grad` variable in the backwards pass at the python level, correct? This is specifically for ROI Pooling.